app/uk/gov/hmrc/personaldetailsvalidation/AssociationMongoRepository.scala

```diff
@@ -97,6 +98,8 @@ class AssociationMongoRepository @Inject() (config: PersonalDetailsValidationMon
 
   }
 
+  def drop: Future[Unit] = collection.drop().toFuture.map(_ => ())
```

Comment:
does this need to be here?, if you need to do this is in the tests you can do app.injector.instanceOf[reponame].collection.removeAll? theres instances of this in FPR and other repos, dropping is quite dangerous as you can drop the indexes and they sometimes don't get re initialised in the tests

Reply:
removed